index.md

```diff
@@ -0,0 +1,76 @@
+## Getting ownCloud up and running
+
+ownCloud is a flexible open source solution for file sharing and synchronization.
+
+The solution consists of the ownCloud server, which runs on Linux; client applications for Microsoft Windows, Mac OS X and Linux; and mobile clients for the Android and Apple iOS operating systems.
+
+This quick start demonstrates how to get a server up and running and enable users to connect to the server using an ownCloud client application.
+
+To do this, we will:
+
+-	[Install and configure an ownCloud server](#installing-and-configuring-the-owncloud-server)
+-	[Create a user account](#creating-a-user-account)
+-	[Connect to the server using a desktop or mobile application](#connect-to-the-owncloud-server-using-a-desktop-or-mobile-application)
+
+## Installing and configuring the ownCloud server
+
+To install the ownCloud server, you need a Linux machine, a web server, a database, and PHP runtime.
+
+We recommend this environment for best performance, stability, support, and full functionality:
+
+Platform	| Recommendation
+-|-
+Operating system | Ubuntu 18.04 LTS
+Database | MariaDB 10+
+Web server | Apache 2.4 with prefork and mod_php
+PHP runtime | 7.4
+
+> **Note:** For other supported environments, see [ownCloud software requirements](https://doc.owncloud.com/server/10.6/admin_manual/installation/system_requirements.html).
+
+You can install ownCloud in multiple ways, including with Docker and Linux Package Manager.
+
+Select an installation method and follow the steps described in the [installation guides](https://doc.owncloud.com/server/10.6/admin_manual/installation/docker/) to install your ownCloud server.
+
```

Comment:
How do I enable users to connect to the Owncloud server using the server's IP address and port 8080?
From the existing documentation and after a looking through the admin console, it was not clear to me. 
Is it the method described here? https://doc.owncloud.com/server/10.6/admin_manual/installation/changing_the_web_route.html.
I couldn't get this to work for me.
Could you please help? Thanks!

Reply:
This is a [post-installation configuration task](https://doc.owncloud.com/server/10.6/admin_manual/installation/manual_installation/manual_installation.html#post-installation-configuration). You need to add the server IP address to the `config.php` file as a trusted domain:

'trusted_domains' => [
   0 => 'localhost',
   1 => 'server1.example.com',
   2 => '192.168.1.50',
],

No need to configure the port 8080 because it is one of the default HTTP ports.